setup.py

```diff
@@ -12,9 +12,10 @@
 dependencies = [
     'numpy>=1.14.5',
     'pandas>=1.1.1',
-    'scipy>=1.1.1',
+    'scipy>=1.7.0',
```

Comment:
why is 1.7 necessary?

The tests should not increase the scipy version. If that is needed for other reasons, I would suggest to change that in an extra PR